### PR TITLE
More functionality

### PR DESCRIPTION
**Pull request by core01**
New options:
- enter / leave - options for animation type
- closeButton - bulma or bootstrap close buttons
- mode - HTML or TEXT, HTML mode enables HTML interpolation in notification body
- permanent - this options makes notification permanent
New methods:
- removeAll - use it to remove all notifications

Also some CSS fixes (margin between notifications) and README.MD improvement

### CODE REVIEW

File: src/Notify.vue [hunk 1]
```diff
@@ -1,9 +1,16 @@
 <template>
-  <div class="notify" :class="'notify-' + options.position">
+  <div class="notify" :class="'notify-' + options.position" :style="{ width: width}">
     <transition-group name="notify" tag="div" @enter="slideDown" @leave="slideUp">
       <div v-for="(item, key) in items" :key="key" class="notify-item">
         <div :class="item.options.itemClass">
-          <span :class="item.options.iconClass" v-if="item.options.iconClass"></span> {{ item.text }}</div>
+          <button v-if="item.options.closeButton === 'bulma'" class="delete" @click="removeItem(key)"></button>
```
Comment (websmurf) [inline src/Notify.vue]: I do not like the strict tie-in to bulma (or bootstrap for that matter), isn't there another solution that will give the same results?

Comment (core01) [reply src/Notify.vue]: Maybe something like that:
```
<button v-if="item.options.closeButtonClass" :class="item.options.closeButtonClass" @click="removeItem(key)" type="button" aria-label="Close"></button>
            <span aria-hidden="true">&times;</span>
          </button>
```
If closeButtonClass option === false (default false) - there is no close button
Would this be good?

Comment (websmurf) [reply src/Notify.vue]: Jup, that would be a lot better :-)

Comment (core01) [reply src/Notify.vue]: @websmurf done

**Comment by core01** [comment]
Hi Adam, please review my PR.

**Review by websmurf — APPROVED** [review]

**Pull request merged**